searx/plugins/__init__.py

```diff
@@ -226,7 +226,10 @@ def plugin_module_names():
 
 
 def initialize(app):
+    # it is possible to block plugins from being registered into plugin chain
+    blocked_plugins = settings['outgoing'].get('blocked_plugins', [])
```

Comment:
After this line https://github.com/searxng/searxng/blob/599d882d0a43b91d6c700edb2f39a900eabbdd5c/searx/settings_defaults.py#L214
add
```python
'blocked_plugins': SettingsValue(list), []),
```

It sets a default value to `[]` and checks that the value is a list.

So here, line 230 in `__init__.py`, you can write `blocked_plugins = settings['outgoing']['blocked_plugins']`